Binary files Mesa-6.5.2.orig/.DS_Store and Mesa-6.5.2/.DS_Store differ
diff -urN Mesa-6.5.2.orig/Makefile Mesa-6.5.2/Makefile
--- Mesa-6.5.2.orig/Makefile	2006-12-02 18:20:04.000000000 +0000
+++ Mesa-6.5.2/Makefile	2007-01-04 16:43:24.000000000 +0000
@@ -23,7 +23,6 @@
 realclean:
 	touch $(TOP)/configs/current
 	$(MAKE) clean
-	-rm -rf lib*
 	-rm -f $(TOP)/configs/current
 	-rm -f `find . -name \*.o`
 	-rm -f `find . -name \*.a`
Binary files Mesa-6.5.2.orig/bin/.DS_Store and Mesa-6.5.2/bin/.DS_Store differ
diff -urN Mesa-6.5.2.orig/bin/mklib Mesa-6.5.2/bin/mklib
--- Mesa-6.5.2.orig/bin/mklib	2006-11-15 14:55:46.000000000 +0000
+++ Mesa-6.5.2/bin/mklib	2007-01-04 16:44:13.000000000 +0000
@@ -560,6 +560,7 @@
             echo "mklib: Making Darwin static library: " ${LIBNAME}
             LINK="ar"
             OPTS="-ruvs"
+	    echo "${LINK} ${OPTS} ${LIBNAME} ${OBJECTS}"
             ${LINK} ${OPTS} ${LIBNAME} ${OBJECTS}
             FINAL_LIBS=${LIBNAME}
         else
@@ -569,10 +570,11 @@
                 OPTS="${ARCHOPT} -bundle -multiply_defined suppress"
             else
 		LIBSUFFIX="dylib"
-                OPTS="${ARCHOPT} -dynamiclib -multiply_defined suppress -current_version ${MAJOR}.${MINOR}.0 -compatibility_version ${MAJOR}.${MINOR}.0 -install_name lib${LIBNAME}.${MAJOR}.${LIBSUFFIX}"
+                OPTS="${ARCHOPT} -dynamiclib -multiply_defined suppress -headerpad_max_install_names -current_version ${MAJOR}.${MINOR}.0 -compatibility_version ${MAJOR}.${MINOR}.0 -install_name @PREFIX@/lib/mesa/lib${LIBNAME}.${MAJOR}.${LIBSUFFIX}"
             fi
-            LINKNAME="lib${LIBNAME}.${LIBSUFFIX}"
-            LIBNAME="lib${LIBNAME}.${MAJOR}.${LIBSUFFIX}"
+            LINKNAME="lib${LIBNAME}.${MAJOR}.${LIBSUFFIX}"
+	    LINKNAME2="lib${LIBNAME}.${LIBSUFFIX}"
+            LIBNAME="lib${LIBNAME}.${MAJOR}.${MINOR}.${LIBSUFFIX}"
 
 	    # examine first object to determine ABI
     	    set ${OBJECTS}
@@ -596,9 +598,11 @@
 	    fi
 
             echo "mklib: Making Darwin shared library: " ${LIBNAME}
+	    echo "${LINK} ${OPTS} -o ${LIBNAME} ${OBJECTS} ${DEPS}"
             ${LINK} ${OPTS} -o ${LIBNAME} ${OBJECTS} ${DEPS}
             ln -s ${LIBNAME} ${LINKNAME}
-            FINAL_LIBS="${LIBNAME} ${LINKNAME}"
+	    ln -s ${LIBNAME} ${LINKNAME2}
+            FINAL_LIBS="${LIBNAME} ${LINKNAME} ${LINKNAME2}"
         fi
         ;;
 

 	  	 
